systemtest/src/main/java/io/strimzi/systemtest/Constants.java

```diff
@@ -59,6 +59,9 @@
     long CO_OPERATION_TIMEOUT_POLL = Duration.ofSeconds(2).toMillis();
     long RECONCILIATION_INTERVAL = Duration.ofSeconds(30).toMillis();
 
+    int GLOBAL_STABILITY_COUNT = 20;
```

Comment:
Might be good to have some comments explaining what these are.

Reply:
I would go for `GLOBAL_RECONCILIATION_COUNT = Duration.ofSeconds(30).toMillis() / GLOBAL_POLL_INTERVAL.` I had also idea to overload method `public static long waitFor(String description, long pollIntervalMs, long timeoutMs, BooleanSupplier ready)` to just to `waitFor(String description, BooleanSupplier ready)` and these two parameters set to some fix value inside parameter `long pollIntervalMs, long timeoutMs`

Reply:
updated )